R/calculate_measures.R

```diff
@@ -64,12 +70,22 @@ calculate_measures <- function(data,
       ))
   }
 
-  pivot_data <- data %>%
-    tidyr::pivot_longer(
-      cols = tidyselect::everything(),
-      names_to = "measure",
-      values_to = "value"
-    )
+
+  if (group_by) {
```

Comment:
What if we make `group_by` `NULL` by default, and then give it `"recid"` for this case?
```suggestion
  if (group_by == "recid") {
```

This will make it slightly more flexible if we wanted to change things in the future.

Happy either way though.